src/Function/Construct/Symmetry.agda

```diff
@@ -0,0 +1,114 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some functional properties are symmetric
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Construct.Symmetry where
+
+open import Data.Product using (_,_; swap)
+open import Function
+open import Level using (Level)
+open import Relation.Binary hiding (_⇔_)
+
+private
+  variable
+    a b c ℓ₁ ℓ₂ ℓ₃ : Level
+    A : Set a
+    B : Set b
+    C : Set c
+
+------------------------------------------------------------------------
+-- Properties
+
+module _ (≈₁ : Rel A ℓ₁) (≈₂ : Rel B ℓ₂)
+         (f : A → B) {f⁻¹ : B → A}
+         where
+
+  inverseʳ : Inverseˡ ≈₁ ≈₂ f f⁻¹ → Inverseʳ ≈₂ ≈₁ f⁻¹ f
+  inverseʳ inv = inv
+
+  inverseˡ : Inverseʳ ≈₁ ≈₂ f f⁻¹ → Inverseˡ ≈₂ ≈₁ f⁻¹ f
+  inverseˡ inv = inv
+
+  inverseᵇ : Inverseᵇ ≈₁ ≈₂ f f⁻¹ → Inverseᵇ ≈₂ ≈₁ f⁻¹ f
+  inverseᵇ (invˡ , invʳ) = (invʳ , invˡ)
+
+------------------------------------------------------------------------
+-- Structures
+
+module _ {≈₁ : Rel A ℓ₁} {≈₂ : Rel B ℓ₂}
+         {f : A → B} {f⁻¹ : B → A}
+         where
+
+  isInverse : IsInverse ≈₁ ≈₂ f f⁻¹ → IsInverse ≈₂ ≈₁ f⁻¹ f
```

Comment:
Any chance we can have `isLeftInverse` and `isRightInverse` as well for completeness's sake?

Reply:
Indeed - done.